bmc-log-collector/testdata/tools/confCSV2Json.XX

```diff
@@ -0,0 +1,87 @@
+package main
```

Comment:
What is this code?

Reply:
that is CSV json converer tool. it will remove near day.

Reply:
chaged the package name